src/identities/mnemonic.ts

```diff
@@ -156,6 +156,10 @@ export default class Mnemonic extends Identity implements IdentityInterface {
     return this.getNextAddress();
   }
 
+  getBlindingPrivateKey(script: Buffer): Buffer {
```

Comment:
lets use hex encoded since this is a public method. This is the standard format used also from `elements-cli dumblindingkey` command.

Reply:
Off course but from the developer point of view, it will need to convert hex to buffer because the script is fetched as Buffer from liquidjs. Maybe add both? string (hex encoded) & buffer ?

Reply:
we need to be consistent how to communicate with "external" world. In the AddressInterface we use the hex encoded, I would keep the same here. 

The reason is also that strings can be encoded anywhere, meanwhile Buffer exists only in the Node world (on the modern browsers is converted to Uint8Array). If we need other clients to interact could become tricky.

Point his let the developer convert to buffer if/when he need.